documentation/KnownChanges.md

```diff
@@ -199,6 +199,10 @@ This rule is disabled by default in StyleCop Analyzers, but can be enabled by us
 :warning: StyleCop Analyzers does not report SA1305 for parameters in overriding methods and methods which implement an
 interface. StyleCop Classic reported SA1305 for all methods.
 
+### SA1313
+
+StyleCop Analyzers treats allows the single and double underscore (`_` and `__`) as lambda parameter names.
```

Comment:
:bulb: It would be good to file a follow-up issue for someone to test if StyleCop Classic allows arbitrary numbers of underscores, and if so, update this section to explicitly state the difference.